pom.xml

```diff
@@ -44,7 +47,34 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>structs</artifactId>
-            <version>1.2</version>
+            <version>1.7</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-step-api</artifactId>
+            <version>2.12</version>
```

Comment:
Was pipeline going to be an optional dependency or not?  

I'm confused about what all has happened here since I built the APIs to let you fetch enclosing blocks for this.

Reply:
Maybe this one just missed an optional?